src/URDFConverter/UrdfUnity/UrdfUnity/FileManager/FileManager.cs

```diff
@@ -0,0 +1,120 @@
+﻿using System;
+using System.IO;
+using UrdfUnity.Urdf.Models;
+using UrdfUnity.Util;
+using UrdfUnity.Parse;
+using NLog;
+
+
+namespace UrdfUnity.FileManager
+{
+    /// <summary>
+    /// Filetypes we care about in the file manager.
+    /// </summary>
+    public enum FileType
+    {
+        UNKNOWN,
+        URDF,
+        XACRO
+    }
+
+
+    /// <summary>
+    /// This class will be handling the FileIO 
+    /// </summary>
+    public class FileManager
+    {
+        Logger LOGGER;
+        
+
+        /// <summary>
+        /// Base constructor, for initializing components
```

Comment:
Nitpick again, but let's follow all the other constructor comments: "Creates a new instance of FileManager"